src/services/UserService.js

```diff
@@ -2,8 +2,43 @@ import { PrismaClient } from '@prisma/client';
 
 const prisma = new PrismaClient();
```

Comment:
Prevenir criação de múltiplos clientes para minimizar conexões com o banco de dados.

[Veja](https://www.prisma.io/docs/guides/performance-and-optimization/connection-management#prevent-hot-reloading-from-creating-new-instances-of-prismaclient).